microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/OpenApiTestConstants.java

```diff
@@ -6,4 +6,5 @@
 public class OpenApiTestConstants {
```

Comment:
These hard-coded constants needs to go away, see other comments about other ways

Reply:
Yes, agree, fixing. Will provide separate commit in this PR following this change.